oarepo_requests/ext.py

```diff
@@ -214,6 +214,13 @@ def init_config(self, app: Flask) -> None:
             if event_type not in app_registered_event_types:
                 app_registered_event_types.append(event_type)
 
+        app_registered_event_types = app.config.setdefault(
+            "NOTIFICATION_RECIPIENTS_RESOLVERS", {}
+        )
+        app.config["NOTIFICATION_RECIPIENTS_RESOLVERS"] = always_merger.merge(
+            app_registered_event_types, config.NOTIFICATION_RECIPIENTS_RESOLVERS
+        )
+
```

Comment:
probably the alwaysmerge is not a correct solution - should replace a value only if it is not already there, not always